packages/core/src/middleware/koa-security-headers.ts

```diff
@@ -103,13 +103,17 @@ export default function koaSecurityHeaders<StateT, ContextT, ResponseBodyT>(
         scriptSrc: [
           "'self'",
           "'unsafe-inline'",
+          "'unsafe-hashes'",
           `${gsiOrigin}client`,
           // Some of our users may use the Cloudflare Web Analytics service. We need to allow it to
           // load its scripts.
           'https://static.cloudflareinsights.com/',
+          // Cloudflare Turnstile
+          'https://challenges.cloudflare.com/turnstile/v0/api.js',
```

Comment:
Should we make this configurable? How about other captcha providers

Reply:
Good idea, we can do it later.